engine/src/flutter/lib/web_ui/lib/src/engine/web_paragraph/code_unit_flags.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class CodeUnitFlags {
```

Comment:
Not sure if this is helpful or not, but might be interesting inspiration here: https://github.com/dart-lang/sdk/blob/8fb9578d95d0ffa55b190ac82dd411412fb566dc/pkg/compiler/lib/src/util/enumset.dart